src/pages/Home.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import { Typography, Button, Fab, AppBar, Toolbar } from "@material-ui/core";
+import { Add } from "@material-ui/icons";
+
+import { PostDialog } from "../components/PostDialog";
+import { PostsList } from "../components/PostsList";
+
+import { auth, firestore } from "../utils/Firebase";
+import { collection, addDoc, Timestamp } from "firebase/firestore";
+import { signOut } from "@firebase/auth";
+
+const postsCollection = collection(firestore, "posts");
+
+export default function Home() {
+  const [postDialogOpen, setPostDialogOpen] = useState(false);
+
+  const post = (postText) => {
+    addDoc(postsCollection, {
+      post: postText,
+      user: auth.currentUser.email,
+      time: Timestamp.now(),
+      image: "" + auth.currentUser.photoURL,
```

Comment:
I was thinking maybe we can remove this field from the firestore document and have a collection with all of the profile photos. What do you think?

Reply:
Yep i was suggesting this earlier but i figured i wouldn't do everything in one pr to keep it reviewable

Reply:
Okay. We can do this now and I'll change this later.